apps/core/scenes/results/Results.js

```diff
@@ -13,25 +15,47 @@ type Props = {|
   +travelTo: string,
   +dateFrom: string,
   +dateTo: string,
+  +returnDateFrom: string,
+  +returnDateTo: string,
 |};
 
 type ResultsType = {|
   +searchItineraries: ResultsListType,
 |};
 
+const statusBarHeight = 20; // @TODO add to orbit design tokens
```

Comment:
Yes I'm sure. 
I tested that on iPhone with rounded display (XS), also on the old one (SE) and also on Android.

We don't have to implement some checks if we use X iphones, because it is solved by using `SafeAreaView` (https://facebook.github.io/react-native/docs/safeareaview)
 in first child of Results screen.

Android also needs to define some top padding and status bar on Android has `20px`. I don't know why they use 0 in your link but when I used `0` padding it looks like this:
<img width="429" alt="screenshot 2019-01-15 at 18 02 29" src="https://user-images.githubusercontent.com/858321/51196591-bede4980-18ef-11e9-8cd0-1dda1c7fe2ac.png">

Reply:
There are issues with `SafeAreaView` check [this](https://github.com/react-navigation/react-navigation/issues/4165)

According to issue the `SafeAreaView` from `react-navigation` is safe to use. But I have also had issues with it. It doesn't always work